src/SDL_ttf.c

```diff
@@ -3125,6 +3127,24 @@ int TTF_SetFontScriptName(TTF_Font *font, const char *script)
 #endif
 }
 
+int TTF_SetFontLanguage(TTF_Font *font, const char *language_bcp47)
+{
+#if TTF_USE_HARFBUZZ
+    TTF_CHECK_POINTER(font, -1);
+
+    if (language_bcp47 == NULL) {
+        font->hb_language = hb_language_from_string("", -1);
+    } else {
+        font->hb_language = hb_language_from_string(language_bcp47, -1);
+    }
+    return 0;
+#else
+    (void) font;
+    (void) language_bcp47;
+    return -1;
```

Comment:
It looks like SDL_ttf does not set errors for unimplemented functionality. Shouldn't it?

```suggestion
    TTF_SetError("unsupported");
    return -1;
```

Reply:
done (1 line + upper case) ! thanks!